ui/admin/app/components/form/role/manage-custom-scopes/index.hbs

```diff
@@ -0,0 +1,110 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if @model.totalItemsCount}}
+  <Rose::Form
+    class='full-width role-manage-custom-scopes-form'
+    @onSubmit={{fn @submit this.allGrantScopes}}
+    @cancel={{@cancel}}
+    @disabled={{@model.role.isSaving}}
+    as |form|
+  >
+    <Hds::SegmentedGroup as |S|>
+      <S.TextInput
+        @value={{@search}}
+        @type='search'
+        placeholder={{t 'actions.search'}}
+        aria-label={{t 'actions.search'}}
+        {{on 'input' @handleSearchInput}}
+      />
+    </Hds::SegmentedGroup>
+    {{#if @model.subScopes}}
+      {{#if this.selectedItems.length}}
+        <div class='table-selected-items-text'>
+          <Hds::Text::Body @tag='p' @color='faint'>
+            {{t
+              'resources.role.scope.titles.selected'
+              items=this.selectedItems.length
+              total=@model.totalItemsCount
+            }}
+          </Hds::Text::Body>
+        </div>
+      {{/if}}
+      <Hds::Table
+        @columns={{array
+          (hash label=(t 'resources.role.scope.title'))
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'resources.role.scope.titles.projects_selected'))
+        }}
+        @model={{@model.subScopes}}
+        @isSelectable={{true}}
+        @onSelectionChange={{this.selectionChange}}
+        @valign='middle'
+      >
+        <:body as |B|>
+          <B.Tr
+            @selectionKey={{B.data.id}}
+            @isSelected={{includes B.data.id this.selectedItems}}
+            @selectionAriaLabelSuffix='row {{B.data.id}}'
+            data-test-org-scopes-table-row={{B.data.id}}
+          >
+            <B.Td>
+              <Hds::Text::Body @tag='p'>
+                {{B.data.displayName}}
+              </Hds::Text::Body>
+            </B.Td>
+            <B.Td>
+              <Hds::Copy::Snippet
+                @textToCopy={{B.data.id}}
+                @color='secondary'
+              />
+            </B.Td>
+            <B.Td>
+              {{! TODO: Add link to projects selected page}}
+            </B.Td>
+          </B.Tr>
+        </:body>
+      </Hds::Table>
+      <Rose::Pagination
+        @model={{array 'global' @model.role.id}}
```

Comment:
Was this the issue causing it to error out when clicking the logo? Was it was from the pagination component?

Just trying to understand what's happening

Reply:
Great questions! Initially i was following the error which led to the pagination component. I think the issue was surrounding the internal ember logic to determine which link is active and determining the wrong link (the one in the pagination comp.) to be the active one. Updating the default route in the rose::header::brand component to be more specific also fixes the issue. I think that could be a better solution ?